core/src/main/java/org/apache/accumulo/core/file/BloomFilterLayer.java

```diff
@@ -281,7 +281,7 @@ static class BloomFilterLoader {
 
     /**
      * Prevent potential CRLF injection into logs from read in user data See
```

Comment:
Super small thing. I saw in all of the other changes you made you added a period  between data and See if it did not have it already. Just for consistency sake I figured I would point it out.

```suggestion
     * Prevent potential CRLF injection into logs from read in user data. See
```

Reply:
Thanks!